common/src/main/java/nl/tudelft/wdm/group1/common/OrdersTopics.java

```diff
@@ -6,8 +6,11 @@
     public static final String ORDER_DELETED = "orderDeleted";
     public static final String ORDER_ITEM_ADDED = "orderItemAdded";
     public static final String ORDER_ITEM_DELETED = "orderItemDeleted";
+    public static final String ORDER_PROCESSED_IN_STOCK_SUCC = "orderProcessedInStockSuccessful";
```

Comment:
I would opt for ORDER_PROCESSED_IN_STOCK_SUCCESSFUL

Reply:
ok